tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -92,7 +92,7 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 
 			// stdOut contains the yaml specification. Store it to a file
 			randomFileName := helper.RandString(6) + ".yaml"
-			fileName := filepath.Join("/tmp", randomFileName)
+			fileName := filepath.Join(os.TempDir(), randomFileName)
```

Comment:
@prietyc123 +1, however i would suggest keep those lines that bring changes to the system in a teardown steps like ```justBeforeEach``` and ```JustAfterEach```. Because, consider a case where you created a file within a spec and did certain operation and an immediate ```oc``` or ```odo``` command fails, then the created file will be left behind for sure. So it's better to have those things in teardown steps, in case the spec command fails the teardown steps will anyway cleanup these files. 

I won't mind if you fix this in a separate pr. But for now atleast remove the created file at the end of the spec.

Reply:
> however i would suggest keep those lines that bring changes to the system in a teardown steps like `justBeforeEach` and `JustAfterEach`.

+1 Good point, its better to accumulate all those such cases in our test suite and fix them in a single shoot, i will take a look and send another in pr as a refactor for sure.

Reply:
/lgtm